docs/core/versions/index.md

```diff
@@ -1,101 +1,226 @@
+
+### Installers
+
+Starting with .NET Core 2.0, downloads for our [daily builds](https://github.com/dotnet/core-setup#daily-builds) and [our releases](https://www.microsoft.com/net/download/core) adhere to a new naming scheme that is easier to understand.
+The installer UI in those downloads were also modified to clearly present the names and versions of the components being installed. In particular, titles now show the same version number that is in the download's file name.
+
+#### File name format
+
+`[product]-[component]-[major].[minor].[patch]-[previewN]-[optional build #]-[rid].[file ext]`
+
+Here are some examples of this format:
+
+```
+dotnet-runtime-2.0.4-macos.10.12-x64.pkg            # Mac runtime installer
+dotnet-sdk-2.0.4-win10-x64.exe                      # Windows SDK installer
+dotnet-sdk-2.0.4-fedora.24-x64.tar.gz               # Fedora 24 binary archive
+
+#Ubuntu file set needed for the SDK
+dotnet-host-2.0.4-ubuntu.16.04-x64.deb              # Host / muxer and host policy
+dotnet-runtime-2.0.4-ubuntu.16.04-x64.deb           # runtime
+dotnet-sdk-2.0.4-ubuntu.16.04-x64.deb               # SDK tools
+```
+
+The format is readable and clearly shows what you're downloading, what version it is, and where you can use it. The runtime package name includes `runtime`, and the SDK includes `SDK`.
+
+#### UI string format
+
+All web site descriptions and UI strings in the installers are kept consistent, accurate, and simple. The following table shows some examples:
+
+| Installer | Window Title                          | Other content in installer | What is installed                               |
+| :--       | :--                                   | :--                        | :--                                             |
+| SDK       | .NET Core 2.0 SDK (x64) Installer     | .NET Core 2.0.4 SDK        | .NET Core 2.0.4 Tools + .NET Core 2.0.4 Runtime |
+| Runtime   | .NET Core 2.0 Runtime (x64) Installer | .NET Core 2.0.4 Runtime    | .NET Core 2.0.4 Runtime                         |
+
+Preview releases differ only slightly:
+
+| Installer | Window Title                                    | Other content in installer        | What is installed                                                   |
+| :--       | :--                                             | :--                               | :--                                                                 |
+| SDK       | .NET Core 2.0 Preview 1 SDK (x64) Installer     | .NET Core 2.0.0 Preview 1 SDK     | .NET Core 2.0.0 Preview 1 Tools + .NET Core 2.0.0 Preview 1 Runtime |
+| Runtime   | .NET Core 2.0 Preview 1 Runtime (x64) Installer | .NET Core 2.0.0 Preview 1 Runtime | .NET Core 2.0.0 Preview 1 Runtime                                   |
+
+It may happen that an SDK release contains more than one version of the runtime. When that happens, the installer UX looks like the following (only the SDK is shown since the Runtime is irrelevant):
```

Comment:
Yes, it can be stripped. The last panel of the installer UX shows the installed versions of the SDK and Runtime.